### PR TITLE
[stable/traefik] Add ability to specify labels for dashboard Ingress

### PR DESCRIPTION
**Pull request by timurkhafizov**
Add ability to add custom labels to the dashboard Ingress resource.

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.13.0
+version: 1.13.1
```
Comment (krancour) [inline stable/traefik/Chart.yaml]: Since this PR adds a feature, it's the minor version number that should be incremented.

**Comment by krancour** [comment]
Could you please also update values.yaml to show inline where/how to specify labels? i.e. similar to what is already done for annotations.

**Comment by timurkhafizov** [comment]
@krancour I've updated PR based on your review 👍

**Comment by krancour** [comment]
LGTM

cc @prydonius @viglesiasce 

This can be merged.

**Pull request merged**